src/components/DealList.tsx

```
@@ -68,9 +71,20 @@ const DealList: React.FC = () => {
     return deals;
   }, [pipelineData]);
 
+  // Calculate unique sales reps
+  const uniqueSalesReps = useMemo(() => {
+    const salesRepsSet = new Set<string>();
+    allDeals.forEach((deal) => {
+      if (deal.sales_rep && deal.sales_rep.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)